solidity/random-beacon/scripts/init-contracts.ts

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable @typescript-eslint/no-unused-expressions, import/no-extraneous-dependencies */
+import { ethers, helpers } from "hardhat"
```

Comment:
Instead of a script, we could make this file a [task](https://hardhat.org/guides/create-task). We could expect the stake owner and operator to be provided as arguments.

For example see https://github.com/keep-network/tbtc-v2/tree/main/solidity/tasks

Reply:
[388f9df](https://github.com/keep-network/keep-core/pull/3026/commits/388f9df64b13b1a3710d0c979e578a7d5fc7f2a9) & [5a48a86](https://github.com/keep-network/keep-core/pull/3026/commits/5a48a8651da2af566d5989040c377573909f9f7f)
I've converted two scripts and made them tasks. However, I'm not sure how much value it adds, to be honest. The same thing we could've accomplished with the existing scripts. As far as I understand tasks are useful when can be used ex. by a third party or can be run standalone. In our case everything is run by `install.sh` and I'm not sure if those tasks will ever be run standalone. Anyway, I might be wrong but just wanted to discuss briefly expectations of what should be put as a "task" and what can be written and run as a "simple script".